corehq/form_processor/submission_post.py

```diff
@@ -54,14 +57,18 @@
 )
 
 
-class FormProcessingResult(namedtuple('FormProcessingResult', 'response xform cases ledgers submission_type')):
+class FormProcessingResponse(namedtuple('FormProcessingResult', 'response xform cases ledgers submission_type')):
     @property
     def case(self):
         assert len(self.cases) == 1
         return self.cases[0]
 
 
 class SubmissionPost(object):
+    class SubmissionFormContext(object):
```

Comment:
why is this an inner class?

Reply:
this was just used within in the class. Its moved out now.